### PR TITLE
Spelling & punctuation

### PR DESCRIPTION
**Pull request by sethkrasnianski**

### CODE REVIEW

File: Documentation/1.0/contribute.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ We want to be as open as possible with our community and we maintain a public [a
 * Submit issues that are general suggestions, as opposed to *specific, technical* bugs or *confirmed* features
 * Create disrespectful or inflammatory comments
 
-> **Note:** This repostiory is for issues only, feature requests will be removed.
+> **Note:** This repositiory is for issues only, feature requests will be removed.
```
Comment (outrunthewolf) [inline Documentation/1.0/contribute.md]: 'repository'

**Comment by outrunthewolf** [comment]
Yikes! Some terrible spelling here. Could you fix the 'repository' and i'll merge it in?

File: Documentation/1.0/contribute.md (hunk 1 above)
Comment (sethkrasnianski) [inline Documentation/1.0/contribute.md]: :open_mouth: thanks

**Comment by sethkrasnianski** [comment]
:+1:

**Comment by outrunthewolf** [comment]
smashing

**Pull request merged**